runtime/onert/backend/trix/KernelGenerator.cc

```diff
@@ -53,6 +56,29 @@ std::unique_ptr<exec::FunctionSequence> KernelGenerator::generate(ir::OperationI
   return ret;
 }
 
+void KernelGenerator::visit(const ir::operation::Bulk &node)
+{
+  using ir::operation::Bulk;
+
+  std::vector<IPortableTensor *> output_tensors;
+  for (auto *ofm_idx : node.getOutputs())
+    output_tensors.emplace_back(_tensor_reg->getPortableTensor(ofm_index));
+
+  std::vector<const IPortableTensor *> input_tensors;
+  for (auto *ifm_idx : node.getInputs())
+    input_tensors.emplace_back(_tensor_reg->getPortableTensor(ifm_idx));
+
+  // parameters
+  const auto binary_path = node.param().binary_path;
+  const auto info_path = node.param().info_path;
```

Comment:
info_path is removed on final PR( #8816 )

Reply:
I will rebase again.